clear-site-data/support/server.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+KNOWN_DATATYPES = ["cookies", "storage", "cache"]
+
+# A support server that receives a list of datatypes in the GET query
+# and returns a Clear-Site-Data header with those datatypes.
+def main(request, response):
+    types = [key for key in request.GET.keys() if key in KNOWN_DATATYPES]
```

Comment:
Not in these particular tests, but eventually yes, so we can remove the restriction.